plugins/command/check_elasticsearch_aws.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python2
+
+from nagioscheck import NagiosCheck, UsageError
+from nagioscheck import PerformanceMetric, Status
+import urllib2
+
+try:
+    import json
+except ImportError:
+    import simplejson as json
+
+HEALTH = {'red':    0,
+          'yellow': 1,
+          'green':  2}
+
+HEALTH_MAP = {0: 'critical',
+              1: 'warning',
+              2: 'ok'}
+
+class ElasticSearchCheck(NagiosCheck):
+    def __init__(self):
+        NagiosCheck.__init__(self)
+
+        self.health = HEALTH['green']
+
+        self.add_option('H', 'host', 'host', "Hostname or network "
+                        "address to probe.  The ElasticSearch API "
+                        "should be listening here.  Defaults to "
+                        "'localhost'.")
+
+        self.add_option('p', 'port', 'port', "TCP port to probe.  "
+                        "The ElasticSearch API should be listening "
+                        "here.  Defaults to 9200.")
+
+    def check(self, opts, args):
+        host = opts.host or "localhost"
```

Comment:
Should this have a default?  Given that it's a check for AWS managed ES, it would never run on localhost.  Should the check throw an error instead (and so be "unknown" in icinga)?